kingfisher_scrapy/spiders/uruguay_historical.py

```diff
@@ -1,35 +1,50 @@
-from kingfisher_scrapy.base_spider import CompressedFileSpider, PeriodicSpider, browser_user_agent
-from kingfisher_scrapy.util import components
+import datetime
 
+import scrapy
 
-class UruguayHistorical(CompressedFileSpider, PeriodicSpider):
+from kingfisher_scrapy.base_spider import CompressedFileSpider
+from kingfisher_scrapy.util import components, handle_http_error
+
+
+class UruguayHistorical(CompressedFileSpider):
     """
     Domain
       Agencia Reguladora de Compras Estatales (ARCE)
     Spider arguments
       from_date
         Download only data from this year onward (YYYY format). Defaults to '2002'.
       until_date
-        Download only data until this year (YYYY format). Defaults to '2017'.
+        Download only data until this year (YYYY format).
```

Comment:
```suggestion
        Download only data until this year (YYYY format).
        If ``from_date`` is provided, defaults to the current year.
```
and similar for `from_date`. You can check the expected docstring by running `scrapy checkall --loglevel=WARNING` command in the project folder as documented in https://kingfisher-collect.readthedocs.io/en/latest/contributing/index.html#commit-the-spider